deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -1036,10 +1051,11 @@ extension ChatViewController: MessagesDataSource {
     }
 
     private func sendImage(url: NSURL) {
-        if let data = try? Data(contentsOf: url as URL) {
-            if let image = UIImage(data: data) {
-                sendImage(image)
-            }
+        if url.pathExtension == "gif" {
+            sendAnimatedImage(url: url)
+        } else if let data = try? Data(contentsOf: url as URL),
+            let image = UIImage(data: data) {
```

Comment:
this is a weird linebreak. it was not obvious to me at a first glance that this line belongs to the if-statement. not a blocker, however, for the future, maybe use one line or align the let-statements horizontally.

Reply:
changed

Reply:
thanks!